app/src/main/java/ch/epfl/sweng/swenggolf/offer/create/CreateHelper.java

```diff
@@ -32,16 +32,27 @@
 import static ch.epfl.sweng.swenggolf.offer.create.CreateOfferActivity.ON;
 import static ch.epfl.sweng.swenggolf.offer.create.CreateOfferActivity.SEPARATION;
 
+/**
+ * Helps the creation of offers.
```

Comment:
If I'm correct, this class is used to help the creation of ShowOfferActivity, not the offers.

Reply:
I think it is used for CreateOfferActivity

Reply:
Yes, that what I wanted to say. By reading the javadoc, we could think it's used for the creation of Offer class. But this is used to help the creation of CreateOfferActivity.